src/test/java/com/karankumar/bookproject/backend/service/PredefinedShelfServiceTest.java

```diff
@@ -1,25 +1,6 @@
-/*
-    The book project lets a user keep track of different books they would like to read, are currently
-    reading, have read or did not finish.
-    Copyright (C) 2020  Karan Kumar
-
-    This program is free software: you can redistribute it and/or modify it under the terms of the
-    GNU General Public License as published by the Free Software Foundation, either version 3 of the
-    License, or (at your option) any later version.
-
-    This program is distributed in the hope that it will be useful, but WITHOUT ANY
-    WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
-    PURPOSE.  See the GNU General Public License for more details.
-
-    You should have received a copy of the GNU General Public License along with this program.
-    If not, see <https://www.gnu.org/licenses/>.
- */
-
```

Comment:
Could you please revert this change?

Reply:
Hi @knjk04 sorry my IDE collapsed these and this one got removed, I've reinstated and also added it to the new Interface files, apologies for this.

Cheers
John